docs/gen.md

```diff
@@ -0,0 +1,297 @@
+# APBS
+
+- apbs_gen
+    - Interactive APBS input file generator. It runs the TnputApp.
+        - Args:
+            - input_file_path: The path to the input file
```

Comment:
Add `## Options` before mentioning all of this

Reply:
> If you can add how to run the interactive apbs and what each key binding too, that would be great

I mean say use `bept gen -a protein.in`, how to use, etc. Any limitations like can you provide `/absolute/path/to/input_file` or it should be present in your directory, etc.